src/components/DayOverview/DayOverviewPdf.tsx

```diff
@@ -0,0 +1,182 @@
+"use client";
+
+import React from "react";
+import { Svg, Document, Page, Text, View, StyleSheet, Path } from "@react-pdf/renderer";
+import { faFlag, faSquare, IconDefinition } from "@fortawesome/free-solid-svg-icons";
+
+import { ParcelOfSpecificDateLocation } from "@/components/DayOverview/DayOverview";
+interface DayOverviewPdfProps {
+    date: Date;
+    location: string;
+    data: ParcelOfSpecificDateLocation[];
+}
+
+interface DayOverviewCardProps extends DayOverviewPdfProps {}
```

Comment:
It shares the same prop as `DayOverviewPdfProp` at this moment
But I want to make it clear like this is just the prop of `DayOverviewCardProps` (don't want to use `DayOverviewPdfProp` as sharing props seems messy
Also it seems more maintainable if other people want to extend it to add more props to the overview card function

Reply:
I rethink about the implementation. I think like it's the implementation of Pdf and Card that coincidentally shares the same props, not like they are supposed to have the same props. Hence, I shouldn't even use the term extend, because that implies it must have everything that pdf has.

Usually we have one prop per function - seem to make things clearer in general